TombEngine/Objects/TR3/Entity/tr3_trex.cpp

```diff
@@ -35,7 +36,15 @@ namespace TEN::Entities::Creatures::TR3
 		TREX_STATE_DEATH = 5,
 		TREX_STATE_ROAR = 6,
 		TREX_STATE_ATTACK = 7,
-		TREX_STATE_KILL = 8
+		TREX_STATE_KILL = 8,
+		
+		// TODO: TR3 states.
```

Comment:
Does this commented blog mean that TR3 T-Rex anims/states are not yet supproted?

Reply:
Animations are there, but the code is not. I did not dig on it, thinking that maybe in the future can create a branch for the TRex and work on those missing states properly. But I didn't see harm in note it there